server/Dockerfile.accel

```diff
@@ -0,0 +1,54 @@
+FROM node:18-bookworm as builder
+
+WORKDIR /usr/src/app
+
+RUN --mount=type=cache,target=/var/cache/apt apt-get update && apt-get install -yqq ffmpeg libraw-dev libjxl-dev libvips-dev libheif-dev
+# debian build for imagemagick has broken RAW support, so build manually
+RUN wget https://imagemagick.org/archive/ImageMagick.tar.gz \
```

Comment:
Do we always want to use the latest released version?

Reply:
We don't particularly need to, but if we set it to a certain version then we need to keep track of updating it. The AIO image seems to have envs for targeting a version and assigns the latest version to them if not specified. I think something like that would be good.

Reply:
I ended up setting target versions for both since it feels more stable that way. Plus it means we can use sha256 checksums so it's safer.